nidaqmx/system/system.py

```diff
@@ -147,7 +147,10 @@ def _update_version(self):
         """
         val = ctypes.c_uint()
 
-        cfunc = lib_importer.windll.DAQmxGetSysNIDAQUpdateVersion
+        try:
+            cfunc = lib_importer.windll.DAQmxGetSysNIDAQUpdateVersion
+        except DaqFunctionNotSupportedError:
+            cfunc = lib_importer.windll.DAQmxGetSysNIDAQMinorVersion
```

Comment:
can we just return 0 here?

it's strange to have the update part of the version in sync with the minor, i.e. 8.0.0, 8.1.1, 8.2.2 (without actual 8.2.0 or 1). 

I'm thinking if there's no update, either have the DriverVersion return major.minor, or major.minor.0, and I guess it's safer for comparisons and so on to use a 0 there.

Reply:
Agreed. I have updated to PR.